Dockerfile

```
@@ -1,14 +1,17 @@
 # build gno
 FROM        golang:1.22 AS build-gno
-RUN         mkdir -p /opt/gno/src /opt/build
+RUN         mkdir -p /opt/gno/src /opt/build /opt/build/contribs/gnodev
```

```suggestion
RUN         mkdir -p /opt/gno/src /opt/build
```

I think it is unnecessary.

---

Yes you're right, I forgot to remove it after a rebase. Thanks for catching this.